.github/workflows/main.yml

```diff
@@ -15,7 +15,7 @@ jobs:
             - uses: actions/checkout@v3
             - uses: actions/setup-node@v3
               with:
-                  node-version: 16
+                  node-version: 18
```

Comment:
would suggest to bump to `20` as it is the LTS

Reply:
@vobu isn't `18` LTS [as well](https://endoflife.date/nodejs)? 🤔